modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -244,15 +244,15 @@ spec:
         dhcp: true
         auto-route-metric: 48 <7>
         address:
-        - ip: "169.254.169.2"
+        - ip: "169.254.0.2"
           prefix-length: 29
       ipv6:
         enabled: true
         dhcp: true
         auto-route-metric: 48
         address:
-        - ip: "fd69::2"
-        prefix-length: 125
+        - ip: "fe80::/64"
```

Comment:
This is not correct. You can't use a link local address as the masqerade IP. It's also probably not /64, although I need to verify that.